### ISSUE TITLE
EnterprisePolicyAsCode module - Don't use unapproved verbs

### ISSUE THREAD

**Issue by o-l-a-v** [Azure/enterprise-azure-policy-as-code#584]
**Is your feature request related to a problem? Please describe.**

Importing EnterprisePolicyAsCode v10.1.2 generates a warning:

```
WARNING: The names of some imported commands from the module 'EnterprisePolicyAsCode' include unapproved verbs that might make them less discoverable. To find the commands with unapproved verbs, run the Import-Module command again with the Verbose parameter. For a list of approved verbs, type Get-Verb.
```

**Describe the solution you'd like**

Follow PowerShell guidelines, use approved verbs.

* <https://learn.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7.4>

**Describe alternatives you've considered**

Surpress the warning stream like so:

```pwsh
Import-Module -Name 'EnterprisePolicyAsCode' 3>$null
```

**Additional context**

* PowerShell v7.4.2
* EnterprisePolicyAsCode v10.1.2

**Comment by anwather** [comment]
These are just guidelines - however will raise with the team.

**Comment by o-l-a-v** [comment]
It also produces noise when importing the module.

**Comment by anwather** [comment]
Yes - someone has picked it up and will fix for a future release.

**Comment by anwather** [comment]
@apybar - as discussed are you ok to update these commands?

**Comment by apybar** [comment]
@anwather  - I'll update these commands

**Comment by apybar** [comment]
This has been resolved in v10.1.5
Documentation has also been updated here: https://azure.github.io/enterprise-azure-policy-as-code/operational-scripts/
